src/i18n.js

```diff
@@ -0,0 +1,33 @@
+import i18n from 'i18next'
+import LanguageDetector from 'i18next-browser-languagedetector'
+import {initReactI18next} from 'react-i18next'
+import XHR from "i18next-http-backend"
+import EnglishTranslations from './Localization/en/translations.json'
+
+i18n
+.use(XHR)
+.use(LanguageDetector)
+.use(initReactI18next)
+.init({
+    resources: {
+        en: EnglishTranslations
+    },
+    fallbackLng: "en",
+    detection: {order: ['navigator', 'cookie', 'localStorage', 'sessionStorage', 'querystring', 'htmlTag']},
+    debug: true,
```

Comment:
Should this be `false` in production mode? (we have code somewhere to check whether we're in webpack prod or dev mode).